lib/v2/freebuf/index.js

```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { type = 'web' } = ctx.params;
+
+    const rootUrl = 'https://www.freebuf.com';
+    const currentUrl = `${rootUrl}/articles/${type}`;
+
+    const options = {
+        headers: {
+            'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36',
+            referer: String(currentUrl),
+            accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7',
+        },
+    };
+
+    const response = await got.get(currentUrl, options);
+
+    const $ = cheerio.load(response.data);
```

Comment:
The web page has an API of `https://www.freebuf.com/fapi/...` and you're suggested to use it instead of going through HTML as said in the [docs ](https://docs.rsshub.app/joinus/new-rss/before-start.html#huo-qu-shu-ju)

Reply:
Before writing this route, I have tried to search for a api of `www.freebuf.com` but nothing found. The api `https://www.freebuf.com/fapi/...` you have mentioned was not found too. If you know where to get any information of `https://www.freebuf.com/fapi/`, please tell me.😃

Reply:
There's an XHR when you click on those categories like 
`https://www.freebuf.com/fapi/frontend/category/list?name=web&page=1&limit=20&select=0&order=0&type=category` and `https://www.freebuf.com/fapi/frontend/category/list?name=es&page=1&limit=20&select=0&order=0&type=category`

Reply:
tks , I will rewriting this route